repo/upgrade_lock_test.go

```diff
@@ -0,0 +1,92 @@
+package repo_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kopia/kopia/internal/repotesting"
+	"github.com/kopia/kopia/repo"
+	"github.com/kopia/kopia/repo/content"
+)
+
+func TestFormatUpgradeSetLock(t *testing.T) {
+	ctx, env := repotesting.NewEnvironment(t, content.FormatVersion1)
+
+	l := &content.UpgradeLock{
+		OwnerID:                "upgrade-owner",
+		CreationTime:           env.Repository.Time(),
+		AdvanceNoticeDuration:  0,
+		IODrainTimeout:         repo.DefaultRepositoryBlobCacheDuration * 2,
+		StatusPollInterval:     repo.DefaultRepositoryBlobCacheDuration,
+		Message:                "upgrading from format version 2 -> 3",
+		MaxPermittedClockDrift: repo.DefaultRepositoryBlobCacheDuration / 3,
```

Comment:
Would it be more correct to use
```go
fmtBlobCacheDuration := env.Repository.ClientOptions().FormatBlobCacheDuration
```
instead of exporting `DefaultRepositoryBlobCacheDuration`?

Reply:
Good point. I am not sure whether client `FormatBlobCacheDuration` is set in this test env. Will check.